exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -138,7 +138,10 @@ drill.exec: {
     class: "org.apache.drill.exec.store.sys.store.provider.ZookeeperPersistentStoreProvider",
     local: {
       path: "/tmp/drill",
-      write: true
+      inmemory: {
+        write: false,
+        capacity: 1000
```

Comment:
Remove this line, otherwise `if (config.hasPath(ExecConstants.SYS_STORE_CAPACITY_PROFILES))` is always true.